python/lsst/ap/association/packageAlerts.py

```diff
@@ -63,6 +67,18 @@ class PackageAlertsConfig(pexConfig.Config):
         default=os.path.join(os.getcwd(), "alerts"),
     )
 
+    doProduceAlerts = pexConfig.Field(
+        dtype=bool,
+        doc="Turn on alert production to kafka if true and if confluent_kafka is in the environment.",
+        default=False,
+    )
+
+    doWriteAlerts = pexConfig.Field(
+        dtype=bool,
+        doc="Write alerts to disk if true.",
+        default=True,
```

Comment:
```suggestion
        default=False,
```

This should default to false to match the existing behavior.

Also, I just appreciated that we need to formally deprecate `doPackageAlerts` rather than simply removing it: see https://developer.lsst.io/stack/deprecating-interfaces.html#config-deprecation

also needs an RFC: https://developer.lsst.io/communications/rfc.html#standard-procedures-that-require-an-rfc

Reply:
If we wanted to avoid the song and dance we could just continue to use `doPackageAlerts` in place of `doWriteAlerts`.  It's less clear but less hassle.

Reply:
I'll go this way for now.

Reply:
Looking at the package again, I think we can actually leave `doWriteAlerts` as `doPackageAlerts` is actually at the diaPipe level and not in the `PackageAlertsTask`? Do `doPackageAlerts` gets set in dia pipe, then inside there `doWriteAlerts` or `doProduceAlerts` gets set. There was never a config to turn on or off writing the alerts out.